yt_dlp/extractor/jtbc.py

```diff
@@ -0,0 +1,149 @@
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        file_id = self._search_regex(r'data-vod="(VO\d+)"', webpage, 'vod id')
+
+        metadata = self._download_json(
+            f'https://now-api.jtbc.co.kr/v1/vod/detail?vodFileId={file_id}',
+            video_id, note='Downloading mobile details', fatal=False)
+
+        playback_data = self._download_json(
+            f'https://api.jtbc.co.kr/vod/{file_id}', video_id, note='Downloading VOD playback data')
+
+        subtitles = {}
+        for sub in traverse_obj(playback_data, ('tracks', lambda _, v: v['file'])):
+            subtitles.setdefault(sub.get('label', 'und'), []).append({'url': sub['file']})
+
+        formats = []
+        for format_id, stream in traverse_obj(playback_data, ('sources', 'HLS', lambda _, v: v['file'], {dict.items}, ...)):
+            m3u8_url = re.sub(r'/playlist(?:_pd\d+)?\.m3u8', '/index.m3u8', stream['file'])
+            formats.extend(self._extract_m3u8_formats(
+                m3u8_url, video_id, m3u8_id=format_id, fatal=False))
+
+        return {
+            'id': video_id,
+            **traverse_obj(metadata, ('vodDetail', {
+                'title': 'vodTitleView',
+                'series': 'programTitle',
+                'age_limit': ('watchAge', {int_or_none}),
+                'release_date': ('broadcastDate', {lambda x: re.match(r'\d{8}', x.replace('.', ''))}, 0),
+                'description': 'episodeContents',
+                'thumbnail': ('imgFileUrl', {url_or_none}),
+            })),
+            'duration': parse_duration(playback_data.get('playTime')),
+            'formats': formats,
+            'subtitles': subtitles,
+        }
+
+
+class JTBCProgramIE(InfoExtractor):
+    IE_NAME = 'JTBC:program'
+    _VALID_URL = r'https?://(?:vod\.jtbc\.co\.kr/program|tv\.jtbc\.co\.kr/replay)/(?P<id>pr\d+)/(?:replay|pm\d+)/?(?:$|[?#])'
+
+    _TESTS = [{
+        'url': 'https://tv.jtbc.co.kr/replay/pr10010392/pm10032710',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'pr10010392',
+        },
+        'playlist_count': 398,
+    }, {
+        'url': 'https://vod.jtbc.co.kr/program/pr10011491/replay',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'pr10011491',
+        },
+        'playlist_count': 59,
+    }]
+
+    def _real_extract(self, url):
+        program_id = self._match_id(url)
+
+        vod_list = self._download_json(
+            'https://now-api.jtbc.co.kr/v1/vodClip/programHome/programReplayVodList', program_id,
+            note='Downloading program replay list', query={
+                'programId': program_id,
+                'rowCount': '10000',
+            })
+
+        entries = [self.url_result(f'https://vod.jtbc.co.kr/player/program/{video_id}', JTBCIE, video_id)
+                   for video_id in traverse_obj(vod_list, ('programReplayVodList', ... 'episodeId'))]
```

Comment:
oops, I think this was my typo
```suggestion
                   for video_id in traverse_obj(vod_list, ('programReplayVodList', ..., 'episodeId'))]
```